helmfile/charts/node-local-dns/templates/node-local-dns.yaml

```
@@ -186,3 +180,23 @@ spec:
           items:
             - key: Corefile
               path: Corefile.base
+
```

Since this is another object it should be a separate document (i.e. separated with `---`)
```suggestion
---
```